youtube_dl/extractor/crunchyroll.py

```diff
@@ -5,6 +5,7 @@
 import json
 import base64
 import zlib
+import cfscrape
```

Comment:
I create a check in _login that display a message if we use login function and that cscrape is not installed.
I don't find how completely quit the download after displaying the message, if you have some tips I will do the modifications.